app/main.py

```diff
@@ -15,5 +15,7 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
```

Comment:
The `collection_of_coins` variable is categorized as mutable, but sets in Python are mutable. Consider whether this categorization aligns with your intended logic.